backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/MetadataUtils.java

```diff
@@ -144,7 +144,7 @@ protected static void init(DSLContext j) {
                   "DROP POLICY IF EXISTS {0} ON {1}",
                   name(SCHEMA_METADATA.getName() + "_POLICY"), SCHEMA_METADATA);
               jooq.execute(
-                  "CREATE POLICY {0} ON {1} USING (pg_has_role(CONCAT({2},UPPER({3}),'/Viewer'),'MEMBER'))",
```

Comment:
add note to PR that explains before pr schema name was case insensitive and converted to uppercase and now after pr is case sensitive. And suggest a way for users/admins to 'fix' there deployments ( is there a way to fix things without having admin access to the pg dm ? ) .

Reply:
Minimally, we would then need to rename the roles. Would be something like query below:
`
do
$$
DECLARE temprow RECORD;
DECLARE nm VARCHAR;
BEGIN 
	FOREACH nm in array array['Viewer','Manager','Owner','Editor'] LOOP
		BEGIN
			FOR temprow IN select distinct r.rolname as oldname, 'MG_ROLE_'||s.table_schema||'/'||nm as newname FROM pg_catalog.pg_roles r join "MOLGENIS"."schema_metadata" s ON(r.rolname ILIKE 'MG_ROLE_'||s.table_schema||'/'||nm||'%') 
			LOOP
				IF NOT EXISTS(SELECT FROM pg_catalog.pg_roles WHERE rolname = temprow.newname) THEN
					RAISE NOTICE 'Value: %', temprow.oldname || ' newname ' ||temprow.newname;
					EXECUTE 'ALTER ROLE "'||temprow.oldname||'" RENAME TO "'||temprow.newname||'"';
				END IF;
			END LOOP;
		END;
	END LOOP;
END;
$$
`

In addition we must then update any RLS policies that have been created that use these. Might be more tricky